app/src/main/java/com/android/calendar/event/EditEventView.java

```diff
@@ -1664,7 +1664,7 @@ public void onTimeSet(View view, int hourOfDay, int minute) {
 
     private void showDatePickerDialog(View view) {
         MaterialPickerOnPositiveButtonClickListener<Long> materialPickerOnPositiveButtonClickListener = timePicked -> {
-            Calendar calendar = Calendar.getInstance();
+            Calendar calendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
```

Comment:
Should this be the local timezone? I don't know what the behaviour was earlier, but if it was the same, it should be okay.